headers/types/dungeon_mode/dungeon.h

```diff
@@ -19065,20 +18944,20 @@ struct dungeon {
     undefined field_0x19901;
     undefined field_0x19902;
     undefined field_0x19903;
-    undefined field_0x19904;
+    undefined field_0x19904; // Unknown entity pointer to a monster.
```

Comment:
If this is an entity pointer, does it make sense to change this to `struct entity*`?